code/modules/admin/tabs/event_tab.dm

```diff
@@ -378,6 +378,33 @@
 
 	give_medal_award()
 
+/client/proc/award_jelly()
+	if(!check_rights(R_ADMIN))
+		return
+
+	// Mostly replicated code from observer.dm.hive_status()
+	var/list/hives = list()
+	var/datum/hive_status/last_hive_checked
+
+	var/datum/hive_status/hive
+	for(var/hivenumber in GLOB.hive_datum)
+		hive = GLOB.hive_datum[hivenumber]
+		if(hive.totalXenos.len > 0)
```

Comment:
Probably want this to be totalDeadXenos or similar. If a hive is destroyed we still want an admin to be able to give a medal hypothetically.